pulp_rpm/app/modulemd.py

```diff
@@ -85,11 +85,13 @@ def parse_modulemd(module_names, module_index):
             modulemd[PULP_MODULE_ATTR.ARTIFACTS] = s.get_rpm_artifacts()
 
             dependencies_list = s.get_dependencies()
```

Comment:
Could you rename some of these variables so that it's more clear what is going on?

```s``` -> ```stream```

```deps``` -> ```platform_deps```

```d_list``` -> ```depmodules_list```

```dependency``` -> ```depmod```